src/SpecFlow.NetCore/AppConfig.cs

```diff
@@ -52,16 +54,34 @@ public static AppConfig CreateIn(DirectoryInfo directory)
 			return config;
 		}
 
-		public void Validate()
+		private static string GuessUnitTestProvider(DirectoryInfo directory)
```

Comment:
Yes, a good point that you can force the provider after the `app.config` is generated. I'm happy enough with guessing for now. Can always change it later if anyone complains about unneeded dependencies - I suspect most people will be using `Newtonsoft.Json` already!

The path argument is vestigial and can be removed, or replaced with a provider argument in the future if required.